tutorials/io/background_loading.rst

```diff
@@ -181,12 +183,14 @@ loader.
         # Update your progress bar?
         get_node("progress").set_progress(progress)
 
-        # ... or update a progress animation?
+        # ...or update a progress animation?
```

Comment:
Not sure about this change, `... continuation` seems more correct to me.

Reply:
I looked a few online resources on style of ellipses, and dang, it's far from clear nor consensual... Apparently some style guides even want it styled as ` . . . ` (with spaces everywhere)...

Reply:
I write it `...like this` myself.